cmd/tar2rpm/main.go

```diff
@@ -83,6 +85,10 @@ func main() {
 		flag.Usage()
 		os.Exit(2)
 	}
+	var buildTimeStamp time.Time
+	if *buildTime != 0 {
```

Comment:
Is this check really necessary, or can it just be condensed to:

```
buildTimeStamp := time.Unix(*buildTime, 0)
```

Reply:
No, surprisingly (as explained in the bug), the zero value of `time.Time` is not the unix epoch. It is the AD 1 year https://en.wikipedia.org/wiki/AD_1 and this ends up as 2042. I'd rather not introduce another zero value here.